packages/sessions-sdk-react/src/session-button.tsx

```diff
@@ -57,17 +55,12 @@ import {
   useTokenAccountData,
 } from "./use-token-account-data.js";
 
+const ONE_SECOND_IN_MS = 1000;
+const ONE_MINUTE_IN_MS = 60 * ONE_SECOND_IN_MS;
+const ONE_HOUR_IN_MS = 60 * ONE_MINUTE_IN_MS;
+const ONE_DAY_IN_MS = 24 * ONE_HOUR_IN_MS;
```

Comment:
how do you feel about these being duplicated in session-limits.tsx

Reply:
I use these all the time lol, I never really felt it was worth centralizing them and exporting them from somewhere since they're just so trivial